notebook/flux.ipynb

```diff
@@ -614,7 +614,7 @@
   },
   {
    "cell_type": "markdown",
-   "id": "42509fce",
+   "id": "13b7ffcb",
```

Comment:
I can't leave a comment at the correct line, but line 668 and similar need a change:

```
"compute_jobreq.cwd = os.path.expanduser(\"~/flux-tutorial/flux-workflow-examples/job-submit-api/\") # set the CWD\n"
```
```

Reply:
Okay that line should be good - will look through our slack thread when I get back for more tweaks. https://github.com/rse-ops/flux-radiuss-tutorial-2023/pull/3/commits/86575eb03f64f7eff350aa0003469c0bc29872ba